.env.dist

```diff
@@ -5,6 +5,7 @@
 ###> symfony/framework-bundle ###
 APP_ENV=dev
 APP_SECRET=dd8bd614c2b1562120629a6264798159
+APP_NAME=symfony-demo-ui
```

Comment:
Can we rename this environment variable -> NEWRELIC_APP_NAME?

Reply:
as discussed, we will use APP_NAME

Reply:
good question! The idea behind is to have a global app name which can be used everythere (for example in monolog etc)